README.md

```diff
@@ -219,7 +219,7 @@ One common use case is to gain Tiptap's rich table editing features while not fu
 Feel free! Be aware that if you add or modify the Vue components and extensions in the npm module itself, you will need to run:
 
 ```
-node app apostrophe-rich-text-widgets:build --npm
+node app apostrophe-tiptap-widgets:build --npm
```

Comment:
Here is the command I had to run to make it work 

When I ran `node app apostrophe-rich-text-widgets:build --npm`, here was the result:

```
There is no such task.

The following tasks are available:

apostrophe-db:reset
apostrophe-caches:clear
apostrophe-migrations:migrate
apostrophe:generation
apostrophe-attachments:rescale
apostrophe-attachments:migrate-to-disabled-file-key
apostrophe-attachments:migrate-from-disabled-file-key
apostrophe-attachments:urls
apostrophe-attachments:recompute-all-doc-references
apostrophe-attachments:reset-uploadfs-permissions
apostrophe-global:generate
apostrophe-pages:unpark
apostrophe-search:index
apostrophe-rich-text-widgets:list
apostrophe-html-widgets:list
apostrophe-video-widgets:list
apostrophe-groups:add
apostrophe-users:add
apostrophe-users:change-password
apostrophe-images:generate
apostrophe-images-widgets:list
apostrophe-files:generate
apostrophe-files-widgets:list
apostrophe-i18n-static:reload
apostrophe-i18n-static:reload-all
apostrophe-i18n-static:generate
apostrophe-workflow:recompute-modified
apostrophe-workflow:dereplicate
apostrophe-workflow:remove
apostrophe-workflow:add-missing-locales
apostrophe-workflow:add-locale-prefixes
apostrophe-workflow:remove-numbered-parked-pages
apostrophe-workflow:resolve-join-ids
apostrophe-workflow:harmonize-workflow-guids-by-parked-id
apostrophe-workflow:diff-draft-and-live
apostrophe-svg-sprites:generate
apostrophe-svg-sprites:import
apostrophe-svg-sprites-widgets:list
apostrophe-tiptap-widgets:build
apostrophe-tiptap-widgets:list
custom:generate
apostrophe-permissions:list

Type:

node app help groupname:taskname

To get help with a specific task.

To launch the site, run with no arguments.
```